dev/io.openliberty.data.internal.persistence/src/io/openliberty/data/internal/persistence/QueryInfo.java

```diff
@@ -1302,7 +1312,11 @@ private Object findAndUpdateOne(Object e, EntityManager em) throws Exception {
                       RepositoryImpl.LIFE_CYCLE_METHODS_THAT_RETURN_ENTITIES);
         }
 
-        return em.merge(toEntity(e));
+        Object returnValue = em.merge(toEntity(e));
+
+        if (trace && tc.isEntryEnabled())
+            Tr.entry(this, tc, "findAndRemoveOne", loggable(returnValue));
```

Comment:
```suggestion
            Tr.entry(this, tc, "findAndUpdateOne", loggable(returnValue));
```

Reply:
Thanks for spotting that and fixing it. I'll accept your correction.